llvm/docs/RISCVUsage.rst

```diff
@@ -330,6 +329,9 @@ The primary goal of experimental support is to assist in the process of ratifica
 ``experimental-smctr``, ``experimental-ssctr``
   LLVM implements the `1.0-rc3 specification <https://github.com/riscv/riscv-control-transfer-records/releases/tag/v1.0_rc3>`__.
 
+``experimental-svukte``, ``experimental-svukte``
```

Comment:
No need to mention ``experimental-svukte`` twice. The one above that I assume you copied is smctr and ssctr.

Reply:
smctr and ssctr are each mentioned 1 time.

Reply:
Oh, I see